lib/SILOptimizer/SILCombiner/SILCombinerMiscVisitors.cpp

```diff
@@ -997,6 +997,31 @@ SILInstruction *SILCombiner::visitIndexAddrInst(IndexAddrInst *IA) {
   return Builder.createIndexAddr(IA->getLoc(), base2, newIndex);
 }
 
+/// Walks over all fields of an aggregate and checks if a reference count
+/// operation for \p value is required. This differs from a simple `isTrivial`
+/// check, because it treats a value_to_bridge_object instruction as "trivial".
+static bool isTrivial(SILValue value, SILFunction *function) {
+  SmallVector<ValueBase *, 32> workList;
+  SmallPtrSet<ValueBase *, 16> visited;
```

Comment:
There is a SmallPtrSetVector now so you don't need to do this

Reply:
No, visited doesn't mean that it's _currently_ in the wordlist. It means: it is or was in the wordlist